pkgs/development/tools/rust/cargo-pgx/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchCrate, rustPlatform, pkg-config, openssl, Security }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-pgx";
+  version = "0.4.5";
+
+  src = fetchCrate {
+    inherit version;
+    crateName = "cargo-pgx";
```

`fetchCrate` supports passing `crateName` as `pname`, so we can use the `inherit` for that as well:

```suggestion
    inherit pname version;
```

---

fixed